src/librustdoc/html/format.rs

```diff
@@ -51,19 +51,20 @@ impl Print for &'_ str {
 crate struct Buffer {
     for_html: bool,
     buffer: String,
+    crate label_id_count: usize,
```

Comment:
`Buffer` seems like the wrong place to put this field. Maybe it should be on `Context` instead?

Reply:
Good point.